packages/rx_bloc_cli/mason_templates/bricks/lib_permissions/__brick__/bin/server/controllers/permissions_controller.dart

```diff
@@ -22,7 +22,8 @@ class PermissionsController extends ApiController {
   Response permissionsHandler(Request request) { {{#has_authentication}}
     final headers = request.headers;
     if (!headers.containsKey(AuthenticationService.authHeader)) {
-      return responseBuilder.buildOK(data: {
+      return responseBuilder.buildOK(data: { {{#enable_feature_auth_matrix}}
+        'AuthMatrixRoute': true,{{/enable_feature_auth_matrix}}
```

Comment:
If the user is not authenticated he should not be able to access `AuthMatrixRoute`